lib/vue-lib/src/design-system/forms/VormInput.vue

```diff
@@ -69,6 +69,12 @@ you can pass in options as props too */
             : null
         "
         class="vorm-input__input-wrap"
+        :class="
+          clsx(
+            'vorm-input__input-wrap',
+           showCautionLines ? themeClasses('bg-caution-lines-light', 'bg-caution-lines-dark') : ''
+          )
+        "
```

Comment:
Implementation of the caution lines, solely used by "unset" in the intrinsic editor now.